CS/vehicalDekho/helper/VehicleDBHelper.java

```diff
@@ -0,0 +1,165 @@
+package helper;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+
+import model.Bike;
+import model.Car;
+import model.Vehicle;
+
+public class VehicleDBHelper {
+
+	private static Connection con = null;
+	private static ResultSet rs = null;
+	private static PreparedStatement ps = null;
+	private static PreparedStatement pNext = null;
+
+	static public boolean isVehicleInsert(Vehicle vehicle) {
```

Comment:
ACTION: Verification for insertion/update are part of test methods and not the APIs therefore you should not keep such methods here. Better test these things directly in the test methods by simply querying (using plane JDBC).